### DISCUSSION TITLE
Where is the detailed description of the angle bracket "<>"?

### DISCUSSION THREAD

**Discussion by mkinit** [vercel/next.js#24490] [Help] [answered]
I saw this empty tag in the "with-redux" example, it solved my problem, but I didn't see the description in the document, and I don't know where it is.(Thanks to Google Translate)

**Comment by apollonian** [comment, marked as answer]
`<></>` is the shorthand syntax for declaring Fragments.
More on this here: https://reactjs.org/docs/fragments.html

**Reply by mkinit** [reply]
thank you very much!
